src/client-apps/nuxeo-drive/nuxeo-drive-metrics-documentation.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Nuxeo Drive metrics
+description: Nuxeo Drive custom metrics documentation.
+tree_item_index: 900
+review:
+  comment: 'release'
+  date: '2021-03-30'
+  status: ok
+toc: true
+---
+
+Starting with Nuxeo Drive 5.1.0 usage metrics have been added using custom HTTP headers. Those metrics are sent to the Nuxeo platform exclusively when the application calls specific operations or endpoints.
+
+On this page, you will find the details about these metrics and their content.
+
+---
+
+Some of these metrics are marked with the `ASYNC` tag, meaning that they are sent in a specific process to avoid spamming the platform with too many requests. More documentation about this can be found in the [asynchronous metrics](#asynchronous-metrics) section.
+
+---
```

Comment:
{{#> callout type='note' }}
Most metrics are sent in real-time, directly attached to the request made against the specific endpoint or operation.

Some of them sent asynchronously to the `/site/api/v1/me` endpoint every 15 minutes (default value). This can be changed with the `custom-metrics-poll-interval` (in seconds) option (experimed in seconds).

That asynchronous process can be disabled through the configuration using the [custom-metrics](https://doc.nuxeo.com/client-apps/nuxeo-drive/#custom-metrics) option.
{{/callout}}

And remove the whole async section at the bottom.